BaragonCore/src/main/java/com/hubspot/baragon/models/BaragonService.java

```diff
@@ -22,7 +22,7 @@
   private final Collection<String> owners;
 
   @NotNull
-  @Pattern(regexp = "/(?:[A-Za-z0-9\\-._~!$&'()*+,;=:@/?]|%[0-9A-Fa-f]{2})*", message = "must be an absolute URL path")
+  @Pattern(regexp = "[a-zA-Z_0-9\\-]+(\\.\\w[a-zA-Z_0-9\\-]+)+(/[#&\\n\\-=?\\+\\%/\\.\\w]+)?|/(?:[A-Za-z0-9\\-._~!$&'()*+,;=:@/?]|%[0-9A-Fa-f]{2})*", message = "must be an absolute URL path")
```

Comment:
i'd almost rather blow away the regex validation here completely, since it's become a mess. i also wonder how this would affect other things that depend on interpreting the `serviceBasePath` value, like wormhole? would it make more sense to create a new field for the domain here?